src/views/dashboard/components/header/header.scss

```diff
@@ -1,168 +1,260 @@
+    padding: 0;
     border-bottom-left-radius: 2rem;
     border-bottom-right-radius: 2rem;
-    position: absolute;
     background-color: $p-color-white;
     width: -webkit-fill-available;
     width: -moz-available;
+    flex-direction: column;
 
-    @media only screen and (max-width: $bp-medium) {
-        padding: 2rem 2.5rem;
-    }
+    &-item {
+      cursor: pointer;
+      font-size: 2rem;
+      border-bottom: 1px solid #ececec;
 
-    ul {
-        padding-left: 2rem;
-        list-style-image:  url('../../../../assets/images/circle.svg');
+      a {
+        color: $p-color-black;
 
-        li {
-            padding-bottom: .5rem;
+        img {
+          margin-right: 0.5rem;
+        }
+
+        &:hover {
+          color: $p-color-black !important;
         }
+      }
+
+      img {
+        margin-right: 1rem;
+      }
+
+      .place-box {
+        padding: 0 2rem;
+      }
     }
 
-    .box {
-        width: 100%;
+    &-item:not(:first-child) {
+      padding: 1.5rem 3rem;
     }
 
-    .country {
-        flex: 0 0 35%;
-        padding: 1rem 5rem 1rem 0;
+    .destination-menu {
+      font-size: 2rem;
 
-        @media only screen and (max-width: $bp-medium) {
-            flex: 0 0 90%;
+      .destination-panel {
+        .ant-collapse-header {
+          padding-left: 4rem !important;
         }
+      }
+    }
 
-        @media only screen and (max-width: $bp-small) {
-            flex: 0 0 100%;
-        }
+    .ant-collapse {
+      border: none !important;
+      background-color: $p-color-white;
 
-        &-name {
-            cursor: pointer;
-            font-size: 2rem;
-            margin: 0 0 1rem;
-            padding-bottom: 1rem;
-            border-bottom: 1px solid $p-color-ec;
-        }
-    
-       .country-wrapper {
-            .region {
-                cursor: pointer;
-                font-size: 1.7rem;
-                margin: 0 0 .5rem;
-            }
-        
-            .place {
-                font-size: 1.5rem;
-                cursor: pointer;
-            }
-       }
+      &-item {
+        border: none !important;
+      }
+
+      &-content {
+        border: none !important;
+      }
+
+      &-header {
+        padding-left: 3rem !important;
+      }
+
+      &-content-box {
+        padding: 0 1.6rem !important;
+      }
     }
+  }
 }
 
 .destination-image {
-    height: 24rem;
-    object-fit: cover;
+  height: 24rem;
+  object-fit: cover;
 
-    @media only screen and (max-width: $bp-large) {
-        height: 20rem;
-    }
+  @media only screen and (max-width: $bp-large) {
+    height: 20rem;
+  }
 
-    @media only screen and (max-width: $bp-medium) {
-        height: 15rem;
-    }
+  @media only screen and (max-width: $bp-medium) {
+    height: 15rem;
+  }
 
-    @media only screen and (max-width: $bp-small) {
-        display: none;
-    }
-}
\ No newline at end of file
+  @media only screen and (max-width: $bp-small) {
+    display: none;
+  }
```

Comment:
can we extract media queries like

@media only screen (max-widht...) {
..all overrides for that screen
.menu { ... }
.menu-mobile {...}
}
same thing for other media